Models/WishListListPart.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Orchard.ContentManagement;
+using Orchard.ContentManagement.Utilities;
+using Orchard.Environment.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Nwazet.Commerce.Models {
+    [OrchardFeature("Nwazet.WishLists")]
+    public class WishListListPart : ContentPart<WishListListPartRecord> {
+        private static readonly char[] separator = new[] { '{', '}', ',' };
+        private readonly LazyField<IEnumerable<ContentItem>> _wishListElements = new LazyField<IEnumerable<ContentItem>>();
+        public LazyField<IEnumerable<ContentItem>> WishListElementsField { get { return _wishListElements; } }
+
+        private string _serializedItemIds {
+            get { return Retrieve(r => r.SerializedIds) ?? ""; }
+            set { Store(r => r.SerializedIds, value); }
+        }
+
+        public int[] Ids {
+            get {
+                return _serializedItemIds
+                    .Split(separator, StringSplitOptions.RemoveEmptyEntries)
+                    .Select(frag => int.Parse(frag))
+                    .ToArray();
+            }
+            set {
+                _serializedItemIds = "{" + string.Join("},{", value) + "}";
```

Comment:
OK, I asked, and he confirmed that the serialization to a string is bad, and he's not doing it any more on more recent code. The braces are there because he's doing `contains` searches on the contents of that field, which in itself is pretty messed up. That should use an index, rather than a search on the contents of the infoset. But here, we're never going to search on that field anyway, so there's no reason to do the weird serialization and braces.